relay-server/src/actors/store.rs

```diff
@@ -199,51 +199,111 @@ impl StoreService {
             }
         }
 
+        let remote_addr = envelope.meta().client_addr().map(|addr| addr.to_string());
+
+        let kafka_messages = Self::extract_kafka_messages(
+            event_item,
+            event_id.ok_or(StoreError::NoEventId)?,
+            scoping,
+            start_time,
+            remote_addr,
+            attachments,
+        );
+
+        for message in kafka_messages {
+            if let Ok(()) = self.produce(topic, scoping.organization_id, &message) {
```

Comment:
Should we log it if self.produce returns an error?

Reply:
Good catch! If `self.produce` fails, the function should early-return with an error, just like it did before:
```suggestion
            self.produce(topic, scoping.organization_id, &message)?;
```

Reply:
i changed it now but I find it a bit weird that we might send half the messages and then abruptly stop if there's an error. Meaning if there is an error with one of the attachments, the other attachments would be produced depending on whether they happen to come before or after the one with the error.